event.py

```diff
@@ -0,0 +1,55 @@
+from copy import deepcopy
+from datetime import datetime, time
+from flask_restful import abort
+
+from eventdao import EventDAO
+
+def read_time(time_str):
+    return datetime.strptime(time_str, '%H:%M:%S').time()
+
+class Event:
+    dao = EventDAO()
+
+    def __init__(self, document: dict):
+        self.id = str(document['_id']) if '_id' in document else None
+        self.name = document['name'] if 'name' in document else None
+        self.days = document['days'] if 'days' in document else None
+        self.from_time = document['from_time'] if 'from_time' in document else None
+        self.to_time = document['to_time'] if 'to_time' in document else None
+        self.description = document['description'] if 'description' in document else None
+
+    @classmethod
+    def get_by_name(cls, event_name):
+        event_dict = cls.dao.get_by_name(event_name)
+        if event_dict is None:
+            abort(404, message='Event {} not found'.format(event_name))
```

Comment:
model should not be running an abort or know anything about status codes, just return None and let the APP handle it